### PR TITLE
DOCS: Update the contribution.md file to show which branch to chekout…

### PR DESCRIPTION
**Pull request by UnkleTayo**
… from

Signed-off-by: Adetayo <akinsanyaadetayo7@gmail.com>

**Description**
This PR updates the contributions file  and it helps point to the right branch to checkout from when starting a new project

This PR fixes issue #1572

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit da14cbfe6888eeb4ac9fb9094f1f7dc535edefef at: https://603f4d416bfd88e3e645bb9f--layer5ng.netlify.app

**Comment by Nikhil-Ladha** [comment]
Could you go through the whole doc and remove all the occurrences of `layer5-ng` and change it to `master` wherever, you think is correct?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Please contribute! The Layer5-ng site uses Gatsby. The process of contributing t
 1. Get a local clone of your fork of the site.
 `git clone https://github.com/layer5io/layer5`
 1. Switch to the `layer5-ng` branch.
-`git checkout upstream/layer5-ng`
+`git checkout upstream/master`
```
Comment (Nikhil-Ladha) [inline CONTRIBUTING.md:59]: It should be only master

Comment (UnkleTayo) [reply CONTRIBUTING.md:59]: Alright, I woll fix that

**Comment by l5io** [comment]
🚀 Preview for commit d3ee8f3e326f12d81ba2275fc61280ea8864823d at: https://603f63e4b8fa0227ba54ead6--layer5ng.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]
Thank you @UnkleTayo for taking this up 👍🏼

**Pull request merged**